autokey-module/README.md

```diff
@@ -0,0 +1,34 @@
+# Google KMS Autokey Terraform Module
+
+Simple Cloud KMS Autokey module that allows creating an initial Autokey configuration.
```

Comment:
Let's link to the public doc on the second instance of "Autokey": https://cloud.google.com/kms/docs/autokey-overview